### PR TITLE
Feature/wmt 636/case progress views

### PR DESCRIPTION
**Pull request by caoimheb**

### CODE REVIEW

File: test/helpers/data/workload-capacity-helper.js [hunk 1]
```diff
@@ -2,109 +2,170 @@ const config = require('../../../knexfile').integrationTests
...
+    return addLdu(inserts)
+  })
+  .then(function (inserts) {
+    return addTeam(inserts)
+  })
+  .then(function (inserts) {
+    return addOffenderManager(inserts)
+  })
+  .then(function (inserts) {
+    return addPopulatedRegion(inserts)
```
Comment (andrew-js-wright) [inline test/helpers/data/workload-capacity-helper.js]: Can we apply this pattern of adding the children when we add a parent organisation?

File: test/integration/services/data/test-get-caseload-progress.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+var omCaseProgressRow = {
+  name: 'Test_Forename Test_Surname',
+  communityLast16Weeks: 10,
+  licenseLast16Weeks: 9,
+  totalCases: 5,
+  warrantsTotal: 30,
+  overdueTerminationsTotal: 30,
+  unpaidWorkTotal: 30
+}
```
Comment (andrew-js-wright) [inline test/integration/services/data/test-get-caseload-progress.js]: Replace these with generator method, as discussed.

File: test/helpers/data/workload-capacity-helper.js (hunk 1 above)
Comment (caoimheb) [inline test/helpers/data/workload-capacity-helper.js]: Updated

File: test/integration/services/data/test-get-caseload-progress.js (hunk 2 above)
Comment (caoimheb) [inline test/integration/services/data/test-get-caseload-progress.js]: Added

**Pull request merged**